docs/getting_started/tutorial.rst

```diff
@@ -0,0 +1,384 @@
+      :alt: Administrative screen
+
+Extend the HomePage model
+-------------------------
+
+Out of the box you get a blank HomePage model and the core app has a
+migration that creates a homepage and configures Wagtail to use it.
+
+This example extends the HomePage model to add a body field.
+
+.. code:: python
+
+    from django.db import models
+
+    from wagtail.wagtailcore.models import Page
+    from wagtail.wagtailcore.fields import RichTextField
+    from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+
+    class HomePage(Page):
+        body = RichTextField(blank=True)
+
+        content_panels = Page.content_panels + [
+            FieldPanel('title', classname="full title"),
+            FieldPanel('intro')
+        ]
+
+``body`` is defined as ``RichTextField``, a special Wagtail field. You
+can use any of the Django core fields. ``content_panels`` define the
+capabilities and the layout of the editing interface. :doc:`More on creating Page models. <../topics/creating_pages>`
+
+
+Run ``python manage.py makemigrations``, then
+``python manage.py migrate`` to update the database with your model
+changes. You must run the above commands each time you make changes to
+the model definition.
+
+To reflect the model changes, edit
+``core/templates/core/home_page.html``. Wagtail uses normal Django
+templates to render each page type. It automatically generates a
+suggestion by separating capital letters with underscores (e.g. HomePage
+becomes home\_page.html).
+
+.. code:: html+django
+
+    {% extends "base.html" %}
+
+    {% load static core_tags wagtailcore_tags %}
+
+    {% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
+
+    {% block content %}
+        {{ self.body|richtext }}
+    {% endblock %}
+
+.. figure:: ../_static/images/tutorial/tutorial_3.png
+   :alt: Updated homepage
+
+A basic blog
+------------
+
+We are now ready to create a blog. To do so run
+``python manage.py startapp blog`` to create a new app in your Wagtail
+site.
+
+The following example defines a basic blog post model in
+``blog/models.py``
+
+.. code:: python
+
+    from django.db import models
+
+    from wagtail.wagtailcore.models import Page
+    from wagtail.wagtailcore.fields import RichTextField
+    from wagtail.wagtailadmin.edit_handlers import FieldPanel
+    from wagtail.wagtailsearch import index
+
+
+    class BlogPage(Page):
+        date = models.DateField("Post date")
+        intro = models.CharField(max_length=250)
+        body = RichTextField(blank=True)
+
+        search_fields = Page.search_fields + (
+            index.SearchField('intro'),
+            index.SearchField('body'),
+        )
+
+        content_panels = Page.content_panels + [
+            FieldPanel('title', classname="full title"),
+            FieldPanel('date'),
+            FieldPanel('intro'),
+            FieldPanel('body', classname="full")
+        ]
+
+Add the new ``blog`` app to ``INSTALLED_APPS`` in
```

Comment:
Maybe put this detail up in the creating the app part above. I think it makes sense to keep those two together